Packs/Hoxhunt/Integrations/HoxhuntV2/HoxhuntV2.yml

```
@@ -0,0 +1,223 @@
+commonfields:
+  id: Hoxhunt v2
+  version: -1
+name: Hoxhunt v2
+display: Hoxhunt v2
+category: Email
+description: "Use the Hoxhunt integration to send feedback to reporters of incidents, set incident sensitivity, and apply SOC classification to incidents."
+configuration:
+- display: Server URL
+  name: url
+  type: 0
+  required: true
+  additionalinfo: e.g. https://api.hoxhunt.com/graphql-external
+  defaultvalue: https://api.hoxhunt.com/graphql-external
+- display: API Key
+  name: api_key
+  type: 4
+  required: true
+  additionalinfo: Input your api key from Hoxhunt
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  type: 19
+  required: false
+  defaultvalue: "1"
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: First fetch timestamp
+  name: first_fetch
+  type: 0
+  required: false
+  additionalinfo: (<number> <time unit>, e.g., 12 hours, 7 days). Defaults to 7 days.
+  defaultvalue: "7 days"
+- display: Fetch limit
+  defaultvalue: "50"
+  name: max_fetch
+  type: 0
+  required: false
+  additionalinfo: Maxium number of incidents per fetch. Cap is 100.
+- display: Only fetch escalated incidents
+  defaultvalue: "false"
+  name: only_escalated_incidents
+  type: 8
+  required: false
+  additionalinfo: "If enabled, only escalated incidents will be fetched. Set up escalation rules in Hoxhunt Response -> Incident Rules"
+- display: Only fetch open incidents
+  defaultvalue: "true"
+  name: only_open_incidents
+  type: 8
+  required: false
+  additionalinfo: "If enabled, only open incidents will be fetched."
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Incident Mirroring Direction
+  name: mirror_direction
+  defaultvalue: Incoming And Outgoing
+  type: 15
+  required: false
+  section: Collect
+  additionalinfo: "Choose the direction to mirror the incident: Incoming (from Hoxhunt to Cortex XSOAR), Outgoing (from Cortex XSOAR to Hoxhunt), or Incoming and Outgoing (from/to Cortex XSOAR and Hoxhunt)."
+  options:
+  - None
+  - Incoming
+  - Outgoing
+  - Incoming And Outgoing
+  hidden:
+  - marketplacev2
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
```

```suggestion
- display: Incident Mirroring Direction
  name: mirror_direction
  defaultvalue: Incoming And Outgoing
  type: 15
  required: false
  additionalinfo: "Choose the direction to mirror the incident: Incoming (from Hoxhunt to Cortex XSOAR), Outgoing (from Cortex XSOAR to Hoxhunt), or Incoming and Outgoing (from/to Cortex XSOAR and Hoxhunt)."
  options:
  - None
  - Incoming
  - Outgoing
  - Incoming And Outgoing
  hidden:
  - marketplacev2
- display: Trust any certificate (not secure)
  name: insecure
  type: 8
  required: false
- display: Use system proxy settings
  name: proxy
  type: 8
  required: false
```
Move the `proxy` to the bottom and removed the "section: Collect" from mirror_direction.
The "section: Collect" should be for related fetch parameters like "IncidentType", "incidentFetchInterval", "first_fetch" etc.